### PR TITLE
Website enhancements

### PR DESCRIPTION
**Pull request by MHerbst**

### CODE REVIEW

**Comment by ThomDietrich** [comment]
I'd suggest to state the [origin of the Vagrantfile](https://github.com/openhab/openhab-docs/blob/gh-pages/Vagrantfile) as a comment, just so changes to one or the other have the chance to be synchronized...

**Comment by MHerbst** [comment]
Thanks for the hint. Added a comment.

File: privacy.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 layout: default
 title: Privacy Policy
 ---
-
+<script>function gaOptout() {document.cookie = disableStr + '=true; expires=Thu, 31 Dec 2099 23:59:59 UTC; path=/';window[disableStr] = true;}</script>
```
Comment (kaikreuzer) [inline privacy.html]: Should this maybe open a message window (alert()) to say that the cookie has been set? ATM, nothing happens if the user clicks the link, which is a bit unexpected.

Comment (MHerbst) [inline privacy.html]: I have just taken the standard coding. But an alert box would be fine. I make a note and will look at it later.

**Pull request merged**